docs/src/pages/components/Text/AndroidText.mdx

```diff
@@ -0,0 +1,50 @@
+---
+platform: android
+documentationId: net.skyscanner.backpack.text
+githubPath: text
+---
```

Comment:
Now with front matter! This is automatically turned into a JS export by a webpack loader, so we can use it in code 💪